app/uk/gov/hmrc/gform/upscan/UpscanController.scala

```diff
@@ -114,8 +114,7 @@ class UpscanController(
                       )
                     case ConfirmationFailure.UpscanFailure(FailureDetails("REJECTED", _)) =>
                       mkFlash(
-                        "file.error.type",
-                        "",
+                        "file.error.rejected.js",
```

Comment:
`file.error.rejected.js` should this rather convey what the error message is about, rather then how is this message triggered. Also please drop`.js` suffix of it.

Reply:
I renamed original file.error.rejected to file.error.invalid.argument as it was only usage was in handling S3 error "InvalidArgument" and at the same time I renamed `file.error.rejected.js` to `file.error.rejected`
By the way we don't have and ACs how to respond to the S3 errors: `InvalidArgument`, `EntityTooLarge` and `EntityTooSmall` . We send `error_action_redirect` to upscan and from what i understood from their README in case of S3 error it will be handled by the call back. Not sure if @ratheeshsm tested that scenario.